lib/handler.ex

```diff
@@ -1,60 +1,65 @@
 defmodule Janus.Handler do
-  alias Janus.Interface
+  @type session_id :: pos_integer
```

Comment:
Shouldn't this be in session module? I've seen session_id_t last time I reviewed session...